scripts/mkcloudhost/cloudfunc

```diff
@@ -10,6 +10,12 @@ function cloudadminnet()
     echo -n 192.168.$((255-2*n))
 }
 
+function cloudironicnet()
+{
+    n=$1
+    echo -n 192.168.$((4*n))
```

Comment:
I think I am fine with it as it is ... but you know, perfect solution would include a check if there isn't some overlap with existing networks... ;-)